src/java.base/share/classes/sun/security/tools/keytool/Main.java

```diff
@@ -1683,7 +1703,9 @@ private void doExportCert(String alias, PrintStream out)
             throw new Exception(form.format(source));
         }
         dumpCert(cert, out);
-        checkWeak(rb.getString("the.certificate"), cert);
+        CertPathConstraintsParameters cpcp =
```

Comment:
Here you could also specify the variant if the cert contains an EKU extension. Same comment applies to other places where you are checking the algorithm constraints of a certificate.

Reply:
Fixed.